File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacPersistance/DAO/FacturasDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface FacturasDAO
    {
        int Insertar(FacturasDTO factura);
        int Modificar(FacturasDTO factura);
        int eliminar(FacturasDTO factura);
        int eliminarLogico(FacturasDTO factura);
        FacturasDTO ObtenerPorId(int facturaId);
        IList<FacturasDTO> ListarTodos();
    }
}

File: Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/FacturasDAO.java
package pe.edu.pucp.softpac.dao;

import java.util.List;
import pe.edu.pucp.softpac.model.FacturasDTO;

public interface FacturasDAO {
    public Integer insertar(FacturasDTO factura);
    
    public FacturasDTO obtenerPorId(Integer facturaId);
    
    public List<FacturasDTO> listarTodos();
    
    public Integer modificar(FacturasDTO factura);
    
    public Integer eliminar(FacturasDTO factura);

    public Integer eliminarLogico(FacturasDTO factura);
    
    public List<FacturasDTO>listarFiltros();
    
}
